src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo.png"
```

Comment:
The `alt` attribute should describe the image content, not just repeat the file name. It helps with accessibility and SEO. For example, if the logo is for 'Moyo', then `alt="Moyo logo"` would be appropriate.